packages/salesforcedx-vscode-expanded/package.json

```diff
@@ -21,8 +21,7 @@
     "vscode": "^1.61.2"
   },
   "scripts": {
-    "vscode:prepublish": "npm prune --production",
```

Comment:
was seeing a weird error generating vsixs for this and the base vscode extension.  There are no deps so removing this step has no impact.